.idea/gradle.xml

```diff
@@ -15,6 +15,7 @@
           </set>
         </option>
         <option name="resolveModulePerSourceSet" value="false" />
+        <option name="useQualifiedModuleNames" value="true" />
       </GradleProjectSettings>
     </option>
   </component>
```

Comment:
Please stop pushing things from the auto-generated idea folder. It's unnecessary

Reply:
I dont know why it keep pushing. I unchecked it so many times.